microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -20,9 +20,7 @@ Remote debugging is necessary for this platform because currently, the built-in
 
 Attach Microsoft Edge DevTools remotely to a WebView2 WinUI 2 (UWP) app as follows:
 
-1.  Install [Microsoft Edge Canary Channel](https://www.microsoftedgeinsider.com/download/canary).
-
-1.  In Microsoft Edge Canary, go to to `edge://inspect` and verify that the **Inspect with Edge Developer Tools** page exists in your version of the browser:
+1.  In Microsoft Edge, go to `edge://inspect`.  The **Inspect with Edge Developer Tools** page opens:
 
     ![DevTools Inspect utility page](./remote-debugging-images/inspect-devtools-page-supported.png)
 

```

Comment:
@maurawinstanley - is this still ok?  WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1

Step 11 in gh rendered section:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/remote-debug/microsoft-edge/webview2/how-to/remote-debugging.md#debugging-desktop-webview2-winui-2-uwp-apps

Reply:
We can remove anything related to setting the release channel preference to 1, this is no longer needed.

Reply:
We can remove anything related to setting the release channel preference to 1, this is no longer needed.

Reply:
my commit that removes setting release channel preference:
https://github.com/MicrosoftDocs/edge-developer/pull/2546/commits/a9953711a02a7ed96176e57624f27fc02f17ab02